indra/llprimitive/llgltfmaterial.cpp

```diff
@@ -208,6 +214,27 @@ std::string LLGLTFMaterial::asJSON(bool prettyprint) const
     return str.str();
 }
 
+// static
+void LLGLTFMaterial::setEmissiveStrengthFromModel(const tinygltf::Material& model, F32& emissive_strength)
+{
+    LL_PROFILE_ZONE_SCOPED;
+
+    const tinygltf::Value::Object &extensions_object = model.extensions;
+    const auto                     emissive_it      = extensions_object.find(GLTF_FILE_EXTENSION_EMISSIVE_STRENGTH);
+
+    LL_INFOS() << "Material extensions: " << model.extensions_json_string << LL_ENDL;
```

Comment:
perhaps `LL_DEBUGS("GLTF")` here? do we expect this to be useful to collect from end users logs in bug reports?